core/server/master/src/main/java/alluxio/master/metastore/caching/CachingInodeStore.java

```diff
@@ -782,4 +852,53 @@ public void removeChild(String name) {
       }
     }
   }
+
+  /**
+   * Cache for inode indices.
+   *
+   * {@link Cache} is used for caching writes to the backing store.
+   * Indices are defined by a composite key of <InodeIndice, InodeId>. Value is
+   * the InodeId itself.
+   *
+   * This cache is not used for reads because:
```

Comment:
Could this be an issue? Each worker queries for the pinlist every second. That would mean every worker will query for the list every second?